### PR TITLE
Changed to py.test instead of nose

### PR DESCRIPTION
**Pull request by noppanit**
#50

### CODE REVIEW

**Comment by noppanit** [comment]
@mpatek #reviewmerge

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 test:
-	nosetests
+	py.test tests/*.py
```
Comment (mpatek) [inline Makefile]: what happens if you just run as `py.test tests`?

Comment (mpatek) [inline Makefile]: @noppanit try renaming all of the .py files in tests/ so that they start with `test_`, then try running just plain `py.test`

**Comment by noppanit** [comment]
@mpatek that did the trick. #reviewmerge again please.

**Comment by mpatek** [comment]
Thanks @noppanit

**Pull request merged**